runningAngle.cpp

```diff
@@ -41,10 +41,13 @@ float runningAngle::add(float angle)
 
 float runningAngle::wrap(float angle)
 {
-  if (_type == DEGREES) angle *= DEG_TO_RAD;
-  while (angle < -PI) angle += TWO_PI;
-  while (angle >= PI) angle -= TWO_PI;
-  if (_type == DEGREES) angle *= RAD_TO_DEG;
+  if (_type == DEGREES) {
+    while (angle < -180) angle += 360;
```

Comment:
is the performance better if you would use 180.0 (float) instead of 180(int)
same for 360

Reply:
tested - IDE1.8.12 => no difference so OK